### DISCUSSION TITLE
How can I config updates of package manager and runtime (eg. pnpm and nodejs)

### DISCUSSION THREAD

**Discussion by oskarkivra** [renovatebot/renovate#30723] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi,

We are using tools like deno, nodejs and pnpm, and have configure the versions with volta/asdf. We are however geting a PR for every new version resulting in a lot of versions locally. How can I configure that we only get an update, let's say every six months. Is it still `packageRules` that I should use or something else?

**Comment by RahulGautamSingh** [comment]
Yes packageRules and schedule

**Reply by oskarkivra** [reply]
I have added
```
{ 
   "matchPackagePatterns": ["pnpm", "nodejs", "deno"], 
   "schedule": ["before 5am on the first day of every 2nd month"] 
}
```
In our config file but we still get PRs for pnpm/nodejs/deno. Can you see any issues with the config?


<details>
<summary>Our full config</summary>

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base", "group:recommended", "schedule:weekly"],
  "hostRules": [...],
  "npmrc": ...,
  "prBodyNotes": [
    "{{#if isMajor}}:warning: MAJOR MAJOR MAJOR :warning:{{/if}}"
  ],
  "minimumReleaseAge": "3 days",
  "labels": ["Renovate"],
  "prConcurrentLimit": 0,
  "automergeType": "branch",
  "packageRules": [
    {
      "matchPackagePatterns": ["^@kivra/"],
      "minimumReleaseAge": "0 days"
    },
    {
      "matchPackagePatterns": ["pnpm", "nodejs", "deno"],
      "schedule": ["before 5am on the first day of every 2nd month"]
    }
  ],
  "vulnerabilityAlerts": {
    "reviewers": [...]
  }
}
```

</details>

**Reply by rarkins** [reply]
Use cron schedules instead
